src/main.zig

```diff
@@ -626,6 +678,7 @@ pub fn main() void { // MARK: main()
 		if(game.world != null) { // Update the game
 			game.update(deltaTime);
 		}
+		Window.Gamepad.update(deltaTime);
```

Comment:
Should be moved into `Window.handleEvents`. Here it's just cluttering the main loop.